Fix shoppingCart & update prodcutImgs  HYD-136 [RTM]

TomasPerez1 opened the pull request.

TomasPerez1 committed:
fix shoppingCart

TomasPerez1 committed:
advance newImg uploader

ramirofazio committed:
Merge branch 'develop' into update_shoppingCart-HYD-136

ramirofazio committed:
fix id on product featured

ramirofazio committed:
feat mobbex integration
Need to connect guast pay

ramirofazio committed:
finish mobbex integration

ramirofazio commented:
@TomasPerez1 

#### - Mobbex integrado, se hicieron cambios en el `shoppingCart` para que funcione todo. Pregunta cualquier cosa que no entiendas :)

> [!WARNING]
> Mobbex anda con la rama `develop` del backend. Asi que mergeate `develop` en la rama de esta tarea asi anda todo

TomasPerez1 committed:
advance

TomasPerez1 committed:
Merge branch 'update_shoppingCart-HYD-136' of https://github.com/ramirofazio/Hydrotek-Web into update_shoppingCart-HYD-136

ramirofazio committed:
finish mobbex integration with repsonse modals

ramirofazio committed:
Merge branch 'update_shoppingCart-HYD-136' of https://github.com/ramirofazio/hydrotek-frontend into update_shoppingCart-HYD-136

ramirofazio committed:
add whatsapp link to modals responses

ramirofazio committed:
fix

ramirofazio committed:
Add CheckoutForm delivery info
localStorage logic to send message
validations

TomasPerez1 committed:
Co-authored-by: Ramiro Fazio Dattoli <ramirofazio@users.noreply.github.com>

TomasPerez1 committed:
finish logic

TomasPerez1 committed:
a

TomasPerez1 committed:
Merge branch 'update_shoppingCart-HYD-136' of https://github.com/ramirofazio/Hydrotek-Web into update_shoppingCart-HYD-136

TomasPerez1 committed:
finishing UI

ramirofazio committed:
remove phone
update text about us

ramirofazio committed:
Merge branch 'update_shoppingCart-HYD-136' of https://github.com/ramirofazio/hydrotek-frontend into update_shoppingCart-HYD-136

TomasPerez1 committed:
finish UI

TomasPerez1 committed:
Merge branch 'update_shoppingCart-HYD-136' of https://github.com/ramirofazio/Hydrotek-Web into update_shoppingCart-HYD-136

TomasPerez1 committed:
cleanup

ramirofazio committed:
finish mobbex and delivery integration:
can't close payResponseModal
form to complete or change deliveryInfo 4 logged user
hot fixes

ramirofazio committed:
Merge branch 'update_shoppingCart-HYD-136' of https://github.com/ramirofazio/hydrotek-frontend into update_shoppingCart-HYD-136

ramirofazio committed:
lint
